v2.1/set-vars.md

```diff
@@ -237,15 +237,16 @@ negative numeric offset from UTC (e.g., `-7`, `+7`). Also, `DEFAULT`,
 
 ## `SET TRACING`
 
-`SET TRACING` changes the trace recording state of the current session. A trace recording can be inspected with the [`SHOW TRACE FOR SESSION`](show-trace.html) statement.
+`SET TRACING` changes the trace recording state of the current session. A trace recording can be inspected with the [`SHOW TRACE FOR SESSION`](show-trace-for-session.html) statement.
 
  Value | Description
 -------|------------
 `off` | Trace recording is disabled.
 `cluster` | Trace recording is enabled; distributed traces are collected.
 `on`  | Same as `cluster`.
-`kv`  | Same as `cluster` except that "kv messages" are collected instead of regular trace messages. See [`SHOW TRACE`](show-trace.html).
+`kv`  | Same as `cluster` except that "kv messages" are collected instead of regular trace messages. See [`SHOW TRACE FOR SESSION`](show-trace-for-session.html).
 `local` | Trace recording is enabled; only trace messages issued by the local node are collected.
+`results` | Result rows and row counts are copied to the session trace. This must be specified explicitly when printing the results produced by a query in the trace is needed.<br><br>Example: `SET tracing = kv, results;`
```

Comment:
I'm not sure I understand this sentence.  Do you mean something like "this must be specified in order for the output of the query to be printed to the session trace" (or?)